src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+
+const employees = document.getElementsByTagName('li');
```

Comment:
Consider using querySelector or querySelectorAll instead of getElementsByTagName for better performance and compatibility.